proj.4/build.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
 ./configure --prefix=$PREFIX --without-jni
+
 make
+
+if [[ $(uname) == Linux ]]; then
+    make check
+fi
```

Comment:
Same error: https://travis-ci.org/SciTools/conda-recipes-scitools/builds/85403903

I am removing that `make check` again.